modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -493,22 +494,41 @@ This request returns a response similar to the following:
 [source,json,linenums]
 ----
 [
- {"timeStamp":"2021-07-01T00:00:00Z","objectStoreRequestCount":0},
- {"timeStamp":"2021-08-01T00:00:00Z","objectStoreRequestCount":38},
- {"timeStamp":"2021-09-01T00:00:00Z","objectStoreRequestCount":28}
+ {"timeStamp":"2023-07-01T00:00:00Z","objectStoreRequestCount":0},
+ {"timeStamp":"2023-08-01T00:00:00Z","objectStoreRequestCount":38},
+ {"timeStamp":"2023-09-01T00:00:00Z","objectStoreRequestCount":28}
+]
+----
+
+To view Object Store usage statistics for the root organization and all sub-organizations for one day, submit a `curl` command similar to the following, 
+replacing ORG_ID with the organization ID for your root organization and BEARER_TOKEN with your bearer token:
+
+[source,console,linenums]
+----
+curl -X GET \
+  'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/ORG_ID?startDate=2023-11-11T17%3A51%3A54.000Z&endDate=2023-11-11T17%3A51%3A54.000Z&period=1month&isMaster=true' \
```

Comment:
We can add a startdate and enddate of 3 days for ex Dec 1 to Dec 3 and the period=1day in the query.